### PR TITLE
Updates to NAM download scripts from UC 2016 fixes.

### PR DESCRIPTION
**Pull request by mfunk**
Found some issues with the NAM  download scripts in MAoW just before presenting at UC. Made a bunch of changes to these two (some for visual aesthetics while presenting) but here they are. **Please merge this one AFTER merging #445.**

### CODE REVIEW

File: suitability/toolboxes/scripts/NAMDownload.py [hunk 1]
```diff
@@ -1,228 +1,170 @@
-# coding: utf-8
 '''
------------------------------------------------------------------------------
-Copyright 2015 Esri
```
Comment (csmoore) [inline suitability/toolboxes/scripts/NAMDownload.py]: Is there a reason the license header was removed?

File: suitability/toolboxes/scripts/NAMDownloadBatch.bat [hunk 2]
```diff
@@ -1,53 +1,19 @@
 ECHO off
-rem -----------------------------------------------------------------------------
```
Comment (csmoore) [inline suitability/toolboxes/scripts/NAMDownloadBatch.bat]: Same question about removing the header

File: suitability/toolboxes/scripts/NAMDownload.py [hunk 3]
```diff
@@ -1,228 +1,170 @@
...
-elif now_time >= time(14, 50, 00) and now_time < time(21, 00, 00):
-    download("1hr12z", "1hr_12z")
-
-elif ((now_time >= time(21, 00, 00) and now_time <= time(23, 59, 59)) or (now_time >= time(00, 00, 00) and now_time <= time(2, 49, 59))):
-    download("1hr18z", "1hr_18z")
+        patternTime = '%H:%M:%S'
+        stringDateNow = datetime.utcnow().strftime(patternDate)
+        stringTimeNow = datetime.utcnow().strftime(patternTime)
+        
+        if now_time >= time(02,50,00) and now_time < time(8,50,00):
```
Comment (csmoore) [inline suitability/toolboxes/scripts/NAMDownload.py]: GitHub highlighting is correctly showing this leading zero as a syntax error in Python 3 (in Python 2 it is interpreted as an octal(base 8) - probably not what is intended either)

File: suitability/toolboxes/scripts/NAMDownload.py (hunk 1 above)
Comment (mfunk) [inline suitability/toolboxes/scripts/NAMDownload.py]: I'm thinking it was something caught in a diff with an earlier non-functioning version.

**Comment by mfunk** [comment]
@csmoore OK, what's next?

File: suitability/toolboxes/scripts/NAMDownloadBatch.bat [hunk 4]
```diff
@@ -1,53 +1,30 @@
...
-
-ECHO(>> %~dp0\DataDownload.txt
-ECHO  -------------------------------------------------------------------------------------------------------------------------------->> %~dp0\DataDownload.txt
 ECHO(>> %~dp0\DataDownload.txt
 ECHO Started NAM data Transfer %Time% %Date%>> %~dp0\DataDownload.txt
 
-python.exe .\NAMDownload.py
 
-timeout /t 60
+CALL python C:\MAOW\Scripts\NAMDownload.py
```
Comment (csmoore) [inline suitability/toolboxes/scripts/NAMDownloadBatch.bat]: Not sure why this was changed to a hard coded path

**Comment by csmoore** [comment]
Sorry - I didn't hear back or realize this was ready to (re)review - can be merged after you confirm the hard-coded path is OK in the .bat

**Comment by mfunk** [comment]
Right now that's a limitation of the MAoW template. In order to have the template work it has to be installed to *C:/MAoW*. If the end user wants the template to work from another location they have to modify several parts of the template, including **NAMDownload.py**. The whole reason we added the NAM files to MAoW is so that we could track changes that were being maded to it (myself for UC and @abrowning80 for AirC2).

**Comment by csmoore** [comment]
We'll assume that is explained somewhere...merging

**Pull request merged**